driver-factory/src/test/java/framework/DriverManagerFactory.java

```diff
@@ -0,0 +1,17 @@
+package framework;
+
+import java.util.HashMap;
+
+public class DriverManagerFactory {
+
+	public static DriverManager getManager() throws Exception {
+		ConfigurationProvider myConfProvider = new ConfigurationProvider();
+		HashMap<String, String> configProps = myConfProvider.getPropertiesFromResourceFile("config.properties");
+		String browser = configProps.get("BrowserType");
+		if (browser.equals("chrome")) {
+			return new ChromeDriverManager();
+		} else {
+			return new EdgeDriverManager();
```

Comment:
Should throw an error on unsupported browserTypes. Don't make EdgeDriver the default